test/index_test.js

```diff
@@ -387,32 +387,32 @@ describe('Client', () => {
     });
 
     describe('getTransactionByHash()', () => {
-      it('should return a transaction binary-encoded if extension is `bin`', async () => {
-        const transaction = await client.getTransactionByHash('b4dd08f32be15d96b7166fd77afd18aece7480f72af6c9c7f9c5cbeb01e686fe', { extension: 'bin' });
+      it('should return a transaction json-encoded by default', async () => {
+        const [{ txid }] = await client.listUnspent();
+        const transaction = await client.getTransactionByHash(txid);
 
-        new Buffer(transaction, 'binary').toString('hex').should.endWith('206e6f7420666f756e640d0a');
+        transaction.should.have.keys('blockhash', 'locktime', 'hash', 'size', 'txid', 'version', 'vin', 'vout', 'vsize');
       });
 
-      it('should return a transaction hex-encoded if extension is `hex`', async () => {
-        const [transaction] = await client.listUnspent();
-        const hex = await client.getTransactionByHash(transaction.txid, { extension: 'hex' });
-
-        hex.should.endWith('cf900000000\n');
-      });
+      it('should match a transaction binary-encoded with hex-encoded', async () => {
```

Comment:
I would keep the same description as the test implementation is not really important in this case.